package.json

```
@@ -66,23 +67,26 @@
     "all-contributors-cli": "^6.24.0",
     "auto": "^10.42.2",
     "esbuild": "^0.16.14",
+    "esbuild-plugin-copy": "^2.1.1",
     "esbuild-plugin-text-replace": "^1.2.0",
-    "eslint": "^8.31.0",
+    "eslint": "8.52.0",
```

```suggestion
    "eslint": "^8.52.0",
```

I don't see any reason to lock the version?